nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/groups/StandardProcessGroup.java

```diff
@@ -4343,6 +4358,18 @@ private void updateProcessGroup(final ProcessGroup group, final VersionedProcess
             group.removeFunnel(funnel);
         }
 
+        for (final String removedVersionedId : inputPortsRemoved) {
```

Comment:
No strong opinion on this, but maybe a log message or a comment would be useful here: "removing the remaining input and output ports".